### ISSUE TITLE
You can order bots to shut down the Outpost Reactor despite the lock

### ISSUE THREAD

**Issue by dylanstrategie** [Regalis11/Barotrauma#3970]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
There are two things you can do to mess with the Outpost reactor that can easily screw up the Outpost pretty bad.

First of all, you can click-drag rods into the reactor. Obviously that will overload the reactor, but the AI seems to take the rods out as long as there's at least one bot nearby to crew the reactor. However, after putting a few rods in, two grenades will cause a nuclear meltdown. I assume you could also distract the people handling the reactor with a fire before putting the rods in to prevent the AI from removing the rods before it's too late

Second of all, you can use Contextual Orders to tell one of your crew to turn off the reactor, which... works. The Outpost AI handling the reactor will turn it back on the moment you unassign them.

**Steps To Reproduce**
- Spawn in Campaign
- Grab your ship's fuel rods
- Slot them into the reactor
- Use two explosives to easily cause a nuclear meltdown
- Alternatively, tell your AI crew to turn off the reactor

**Version**
Barotrauma Unstable V0.10.600.0. Windows 10

**Additional information**
I have heard rumours that you can blow up the station without risking any reputation loss by simply putting rods in and waiting, but I'm not sure if that's possible since the AI intentionally takes the rods out as you put them in and slots them in their inventory. Messing with the reactor's rods still opens it to being exploded though

**Comment by Regalis11** [comment]
Partially fixed in https://github.com/Regalis11/Barotrauma-development/commit/c22efacc83c2c299199d12e77204cd8c8707f54d (you can no longer drag and drop rods into the reactor)

**Comment by itchyOwl** [comment]
I guess we need to disallow those contextual orders. But the fix shouldn't categorically disallow using other types of subs. I'm not entirely sure should we just deny targeting any items in outposts or just particular items? Are there cases where we need to target the outpost items with contextual orders?

**Comment by Regalis11** [comment]
Related: https://github.com/Regalis11/Barotrauma/issues/4126

**Comment by ekqvist** [comment]
Added a new editable property "NonPlayerTeamInteractable" for the Item class in Regalis11/Barotrauma-development@252e7e1 and modified code which references the existing property "NonInteractable" so things should properly take into account the new property. I would suggest also testing various ways of assigning orders as I fixed some issues this change caused with them.

Set the new property to true for outpost reactors to prevent characters on player teams from interacting with them in Regalis11/Barotrauma-development@b3c8cea.
